### DISCUSSION TITLE
Keeping default user and password secret

### DISCUSSION THREAD

**Discussion by kennylajara** [docker-library/rabbitmq#521] [Q&A] [answered]
I created a file `rabbimq.conf` and this is the current content: 
```
loopback_users.guest = false
listeners.tcp.default = 5672
disk_free_limit.relative = 2.0
default_pass = test
default_user = test
```

I don't want to gitignore the this file, but I don't want my credentials to be stored. Ia m trying to use some king of environment variable or something but you deprecated the default username and password environment variables on the last version. 
How can I keep secret my credentials now?

**Comment by yosifkit** [comment, marked as answer]
RabbitMQ 3.9.4 added support for a few variables directly (instead of the overly complex bash script): https://github.com/rabbitmq/rabbitmq-server/releases/tag/v3.9.4 and are allowed though the entrypoint script with https://github.com/docker-library/rabbitmq/pull/519.

(Note that environment variables are not secret from someone on the host or with access to Docker)
